mod/dfrn_request.php

```diff
@@ -463,7 +463,7 @@ function dfrn_request_post(App $a)
 			}
 
 			$url = str_replace('{uri}', $uri, $url);
-			$a->redirect($url);
+			$a->internalRedirect($url);
```

Comment:
I'm pretty sure `$url` is absolute, see line 285.

Reply:
Ah, I was misguided because of line 453 to 463 which looked for me like some logic for a relative path. But in the end it's just for the `str_replace` at the end..